### PR TITLE
Test sign up

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary
Added libraries to test and a new test for user's sign up

## Trello Card
[link](https://trello.com/c/GwuIqCnG/23-tests-de-sign-up)

### CODE REVIEW

**Comment by alanlamaswolox** [comment]
Finally the CI succeed :tada:

File: test/controllers/user.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const supertest = require('supertest');
+const models = require('../../app/models/index');
+const app = require('../../app');
+
+const request = supertest(app);
+
+describe('usersController.signUp', () => {
+  it('Creates a user', () => {
```
Comment (SantiagoFoster) [inline test/controllers/user.test.js]: This one is fine, but you should test for border cases too. I'll let you think about what border cases I'm refering to, there are 3 of them

**Review by SantiagoFoster — CHANGES_REQUESTED** [review]

File: test/controllers/user.test.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  it('Try to create a existing user and fails', () =>
+    request
+      .post('/users')
+      .send(userAttributes)
+      .then(() =>
+        request
+          .post('/users')
+          .send(userAttributes)
+          .then(response => expect(JSON.parse(response.text).internal_code).toBe('email_already_in_use'))
```
Comment (SantiagoFoster) [inline test/controllers/user.test.js]: You should not have to parse `response.text`. Check if you can access the property `internal_code` directly.  Should be in either `response.body` or `response.response.body`

File: test/controllers/user.test.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  password: 'password1923'
+};
+
+describe('usersController.signUp', () => {
+  it('Creates a user', () =>
+    request
+      .post('/users')
+      .send(userAttributes)
+      .then(response => {
+        expect(response.text).toBe('Omar');
```
Comment (SantiagoFoster) [inline test/controllers/user.test.js]: Check the signup PR. After fixing it, this should be calling the first_name property

**Review by SantiagoFoster — CHANGES_REQUESTED** [review]

**Review by SantiagoFoster — APPROVED** [review]

**Pull request merged**